docs/source/markdown/podman-volume-create.1.md

```diff
@@ -129,6 +129,80 @@ All containers are assigned larger project IDs (e.g. >= 100000).
 All volume assigned project IDs larger project IDs starting with 200000.
 This prevents xfs_quota management conflicts with containers/storage.
 
+## MOUNT EXAMPLES
+
+podman volume create allows you to pass the `type`, `device`, and `o` options to `mount(8)` when using the `local` driver.  
+
+Here are some examples of what you can do with that
+
+## [s3fs-fuse](https://github.com/s3fs-fuse/s3fs-fuse)
+
+[s3fs-fuse](https://github.com/s3fs-fuse/s3fs-fuse) or just `s3fs`, is a fuse filesystem that allows you to mount s3 prefixes as filesystem mounts.
+
+**Installing:**
+```shell
+dnf install s3fs-fuse
+# should also install fuse3 as a dependency
+```
+
+**Simple usage:**
+```shell
+s3fs --help
+s3fs -o use_xattr,endpoint=aq-central-1 bucket:/prefix /mnt
+```
+
+**Eqivelent through `mount(8)`**
+```shell
+mount -t fuse.s3fs -o use_xattr,endpoint=aq-central-1 bucket:/prefix /mnt
+```
+
+**Eqivelent through `podman volume create`**
+```shell
+podman volume create s3fs-fuse-volume -o type=fuse.s3fs -o device=bucket:/prefix -o o=use_xattr,endpoint=aq-central-1
+```
+
+**You can then mount it in a container with**
+```shell
+podman run -v s3fs-fuse-volume:/s3:z --rm -it fedora:latest
+```
+
+Please see the available [options](https://github.com/s3fs-fuse/s3fs-fuse/wiki/Fuse-Over-Amazon#options) on their wiki.
+
+### Using with other container users
+
+The above example works because the volume is mounted as the host user and inside the container `root` is mapped to the user in the host. 
+
+If you tried to access the mount with a different user inside the container you will be hit with a "Permission denied"
+```shell
+podman run --user bin:bin -v s3fs-fuse-volume:/s3:z,U --rm -it fedora:latest
+ls /s3
+# ls: /s3: Permission denied
+```
+
+Even though that user should be maped to one of your `subuid`'s  
+
+In FUSE-land mounts are protected to the user who mounted it, if you want other users to be able to access these mounts, you need to specify the `allow_other` mount option.
```

Comment:
In FUSE-land **COMMA** mounts ... mounted it **PERIOD**.

And, our guidelines prevent use of the word "you". Can you please phrase this differently?